util/gctuner/tuner_test.go

```diff
@@ -53,8 +53,8 @@ func TestTuner(t *testing.T) {
 	runtime.GC()
 	for i := 0; i < 100; i++ {
 		runtime.GC()
-		require.GreaterOrEqual(t, tn.getGCPercent(), uint32(50))
-		require.LessOrEqual(t, tn.getGCPercent(), uint32(100))
+		require.GreaterOrEqual(t, tn.getGCPercent(), MinGCPercent)
+		require.LessOrEqual(t, tn.getGCPercent(), uint32(200))
```

Comment:
This should not change? the active heap can be equal or larger than threshold/2, so we should expect the percentage equal or less than 100%.

Also, this test have repeated many hardcode values.

Reply:
updated

Reply:
the origin threshold is so small for the test case to be disturbed by the memory of the test framework.